Changelog.md

```diff
@@ -2,6 +2,14 @@
 
 ### Ignition CMake 2.X.X (20XX-XX-XX)
 
+1. Make the OGRE plugin path discovery portable
+  * [Pull request #182](https://github.com/ignitionrobotics/ign-cmake/pull/182)
+  * [Pull request #101](https://github.com/ignitionrobotics/ign-cmake/pull/101)
+
+1. Fix bad INCLUDE_DIRS cache confusion
+  * [Pull request #182](https://github.com/ignitionrobotics/ign-cmake/pull/182)
+  * [Commit 5320ac6](https://github.com/ignitionrobotics/ign-cmake/commit/5320ac64c23e379562f9abfccf5eea69456cb5c1)
```

Comment:
This was a backport from `ign-cmake2`, so do we need to add it to the changelog of `ign-cmake2` since it's already in `ign-cmake2`? I'm not sure which particular commit in `ign-cmake2` added this because it looks like it was done while we were still on BitBucket, but the content is there if you look at the `ign-cmake2` branch